### ISSUE TITLE
App Crash - ConcurrentModificationException

### ISSUE THREAD

**Issue by jpage4500** [TeamAmaze/AmazeFileManager#854]
not sure exactly how this occurred but I do see and have fixed the issue as described below:

Testing on S7 running 7.0

```
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime     java.util.ConcurrentModificationException
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at java.util.HashMap$HashIterator.nextEntry(HashMap.java:851)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at java.util.HashMap$EntryIterator.next(HashMap.java:891)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at java.util.HashMap$EntryIterator.next(HashMap.java:890)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at com.amaze.filemanager.ui.icons.IconHolder$1.processResult(IconHolder.java:110)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at com.amaze.filemanager.ui.icons.IconHolder$1.handleMessage(IconHolder.java:89)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at android.os.Handler.dispatchMessage(Handler.java:102)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at android.os.Looper.loop(Looper.java:154)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at android.app.ActivityThread.main(ActivityThread.java:6642)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at java.lang.reflect.Method.invoke(Native Method)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
11-30 19:47:49.009    com.amaze.filemanager    27621    -    E    AndroidRuntime         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
11-30 19:47:56.137    com.amaze.filemanager    27621    28545    E    BitmapFactory     Unable to decode stream: java.io.FileNotFoundException: smb:/192.168.0.100/MEDIA/TV/Animal Kingdom (2016)/fanart.jpg (No such file or directory)
```

**Comment by jpage4500** [comment]
PR: https://github.com/TeamAmaze/AmazeFileManager/pull/855

**Comment by EmmanuelMess** [comment]
No need to fix this, Glide would replace IconHolder when build tools 26 are available (#848).
